crates/torii/core/src/query_queue.rs

```diff
@@ -69,20 +91,54 @@ impl QueryQueue {
                 }
             }
 
-            total_affected += tx.execute(query).await?.rows_affected();
+            total_affected += tx
+                .execute(query)
+                .await
+                .with_context(|| format!("Failed to execute query: {}", statement))?
+                .rows_affected();
         }
 
         tx.commit().await?;
 
         while let Some(message) = self.publish_queue.pop_front() {
-            match message {
-                BrokerMessage::ModelRegistered(model) => SimpleBroker::publish(model),
-                BrokerMessage::EntityUpdated(entity) => SimpleBroker::publish(entity),
-                BrokerMessage::EventMessageUpdated(event) => SimpleBroker::publish(event),
-                BrokerMessage::EventEmitted(event) => SimpleBroker::publish(event),
+            send_broker_message(message);
+        }
+
+        while let Some((statement, arguments, query_type)) = self.publish_queries.pop_front() {
+            let mut query = sqlx::query_as(&statement);
+            for arg in &arguments {
+                query = match arg {
+                    Argument::Null => query.bind(None::<String>),
+                    Argument::Int(integer) => query.bind(integer),
+                    Argument::Bool(bool) => query.bind(bool),
+                    Argument::String(string) => query.bind(string),
+                    Argument::FieldElement(felt) => query.bind(format!("{:#x}", felt)),
+                }
             }
+
+            let broker_message = match query_type {
```

Comment:
Does this mean we also fetch the entity for each set entity we do, to get the new EntityUpdated? Let's either make the queries `RETURNING *` and use that or just initialize the EntityUpdated without fetching it

Reply:
@lambda-0x any update on this based on the @Larkooo comment?

Reply:
oh my bad, i made the changes yesterday but forgot to push